sdks/java/core/src/test/java/org/apache/beam/sdk/testing/FileChecksumMatcherTest.java

```diff
@@ -123,7 +122,7 @@ public void testMatcherThatVerifiesFileWithEmptyContent() throws IOException {
     FileChecksumMatcher matcher =
         new FileChecksumMatcher(
             "da39a3ee5e6b4b0d3255bfef95601890afd80709",
-            IOChannelUtils.resolve(tmpFolder.getRoot().getPath(), "*"));
+            tmpFolder.getRoot().toPath().resolve("*").toString());
```

Comment:
The code here is, FWICT, doing exactly what the existing code already does: https://github.com/apache/beam/blob/master/sdks/java/core/src/main/java/org/apache/beam/sdk/util/FileIOChannelFactory.java#L170

The windows issue in FileIOChannelFactory was in the `String`->`Path` code ([`specToFile`](https://github.com/apache/beam/blob/master/sdks/java/core/src/main/java/org/apache/beam/sdk/util/FileIOChannelFactory.java#L74)), but here we start with a `File` so that ought not be an issue.

PTAL?